platforms/jvm/toolchains-jvm/build.gradle.kts

```diff
@@ -50,6 +50,10 @@ dependencies {
     testImplementation(testFixtures(project(":core")))
     testImplementation(testFixtures(project(":logging")))
 
+    testFixturesImplementation(project(":native"))
```

Comment:
❓ it's weird to have this as a dependency. Why is it needed?

Reply:
Due to using `org.gradle.internal.nativeintegration.services.FileSystems` in `JdkRepository.zip`. Maybe there's a smarter way to set the file permissions in the archive?